tools/pipelines/server-routerlicious.yml

```diff
@@ -96,12 +96,28 @@ pr:
     exclude:
     - server/routerlicious/kubernetes/routerlicious
 
+variables:
+  - template: /tools/pipelines/templates/1ES/include-vars.yml@self
+    parameters:
+      releaseImage: true
+      publishOverride: ${{ parameters.publishOverride }}
+      releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
+      buildNumberInPatch: false
+  # Note: we tried to centralize the use of the 'container-registry-info' variable group (i.e. put it inside the shared
```

Comment:
Thanks for the comment! Have we reported this deficiency? It isn't immediately clear to me if this is something we'll be able to clean up in the future.